webapp/views/student.py

```diff
@@ -93,6 +96,58 @@ def submissions(student: Student | None, page: int):
     )
 
 
+@blueprint.route("/home", methods=["GET"])
+@authorize(db.students)
+def default_home(student: Student | None):
+    if config.config.registration and student and student.group is None:
+        return redirect("/")
```

Comment:
```suggestion
    if not config.config.registration or not student or student.group is None:
        return redirect("/")
```
Может быть, так? Если выключена регистрация, то редирект на главную. Если регистрация включена и студент не авторизован, то редирект на главную. Если группа не выбрана, то тоже на главную.

Reply:
Да, исправил в [temp: fix: redirect from /default_home to /](https://github.com/kispython-ru/dta/pull/56/commits/7a35303998c92a18b2781e63109d94dc74a8c08e)